applications/automatic-post-editing.md

```diff
@@ -2,6 +2,47 @@
 nav_order: 5
 parent: Application areas
 title: Automatic post-editing
-description: A method to correct errors in the machine translation output
-layout: coming_soon
+description: Automatic correction of machine translation errors
 ---
+
+**Automatic post-editing** (APE) is the task of correcting automatically the output of a machine translation system.
+Automatic post-editing systems should
```

Comment:
This will not be machine translated easily.  Better to make segments that don't split a sentence, for example:

> Automatic post-editing systems should meet multiple requirements: